### ISSUE TITLE
Any special Build instructions (once completed)?

### ISSUE THREAD

**Issue by BHunter2889** [centrual/cordova-template-framework7-vue-webpack#35]
Just curious if this template requires any special build instructions for deployment? To clarify, this build will work for all Phonegap/Cordova supported platforms correct?  Just asking in advance because I've not built a PhoneGap/Cordova app with this many different frameworks or with Node involved. On a deadline, so I'm trying to make this transition as seamless as possible. Thanks in advance!

**Comment by BHunter2889** [comment]
Also, this template is very different from the other available default PhoneGap/Cordova templates... Any suggestions on how to go about porting my existing project into this would be much appreciated. Experimenting on my own in the meantime, just thought I'd ask, Thanks!

**Comment by centrual** [comment]
Hello @BHunter2889 

First of all this template uses [Cordova](https://cordova.apache.org/), [Framework7](https://framework7.io/), [Vue.js](https://vuejs.org/) and [Webpack](https://webpack.js.org/concepts/).

So you need to know this combo well for make an app with this template.

I made this template because i want to make an app fast with hot push and cordova. So i added a live reload feature (`-- --live-reload` or alias `-- --lr`) to normal cordova run commands for fast development.

This template uses normal cordova, so you do not have to do anything special except what you normally do with cordova.

Most of the time, you need to use `src` folder. So you need to transfer your vue files into the `src` folder. Program automatically manages the `www` folder, so **don't** add any of your files into the `www`.

If you have special requirements (like manifest.json in www directory), i can implement it if you ask. Currently if you add `manifest.json` into the `src` folder, template will copy this file into the `www` on compile time.

In summary
1) Use the `src` folder instead of the `www` folder
2) You can use `-- --lr` command for live reload. ( E.g: `cordova run android -- --lr` )

Everything else is the same as the normal cordova.

Have a great day! 😃

**Comment by centrual** [comment]
Closing the issue.

**Comment by BHunter2889** [comment]
Apologies for not addressing your previous follow up. This was indeed very useful information. I appreciate the time in helping me resolve my issues. Thanks!
